src/openqaoa-core/backends/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Contains backend specific codes for implementing QAOA on supported simulators
+and QPU backends
+
+Currently Supported:
+	Qiskit:
+		QASM Simulator
+		Statevector Simulator
+		IBMQ available QPUs
+	PyQuil:
+		Rigetti QPUs
+		Statevector Simulator
+	Vectorized:
+		Fast numpy native Statevector Simulator
+"""
+from .qaoa_vectorized import QAOAvectorizedBackendSimulator
+from .qaoa_analytical_sim import QAOABackendAnalyticalSimulator
+from .devices_core import DeviceBase, DeviceLocal
```

Comment:
why are we importing 'DeviceBase' here? it is an abstract class

Reply:
Can be removed. Need to check if there is some kind of circular import, which is why its imported there.

Reply:
Resolved in commit [bf015fb](https://github.com/entropicalabs/openqaoa/pull/170/commits/bf015fbaa9ce4e043bcbc4dfe34700a0886feeb0)